### DISCUSSION TITLE
Tailscale script - Installing Tailscale... bash: line 4: wget: command not found

### DISCUSSION THREAD

**Discussion by Manu-65** [community-scripts/ProxmoxVE#625] [General]
Hello, i'm beginner.

I have installed ProxmoxVE in Raspi5.
```
# uname -a
Linux mgs-pimox500 6.6.62+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.62-1+rpt1 (2024-11-25) aarch64 GNU/Linux
```
![imagen](https://github.com/user-attachments/assets/4071b8ea-15f6-43ec-9a9e-004617c9b1dc)

I am preparing my first LXC container (debian 12).
![imagen](https://github.com/user-attachments/assets/a87de93b-a22a-44aa-a452-066d428acf59)

When i execute the script within the Proxmox shell i get the next message:
```
Installing Tailscale... 
bash: line 4: wget: command not found
```

```
# bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/misc/add-tailscale-lxc.sh)"
  ______      _ __                __
 /_  __/___ _(_) /_____________ _/ /__
  / / / __ `/ / / ___/ ___/ __ `/ / _ \
 / / / /_/ / / (__  ) /__/ /_/ / /  __/
/_/  \__,_/_/_/____/\___/\__,_/_/\___/

This will add Tailscale to an existing LXC Container ONLY. Proceed(y/n)? y
```

![imagen](https://github.com/user-attachments/assets/57d06b50-8373-4f88-8b13-545ef271315b)

![imagen](https://github.com/user-attachments/assets/fe74109f-0c0d-4ede-8809-d7427f8f6658)

i verified the contents of /etc/pve/lxc/100.conf
It is matching the information described in  
https://tailscale.com/kb/1130/lxc-unprivileged#faq

Proxmox shell:
```
cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
```

Container shell:
```
cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
```

I get blocked, do you have any idea about?

Raspi5 + Proxmox + Container are incompatibles? 

Regards

**Comment by MickLesk** [comment]
Raspi 5? -> you need the pimox Scripts If is an arm64 device 
Visit https://github.com/asylumexp/Proxmox 

I'm not sure if this is the case with the R5. 

Otherwise, just as it says. Wget is missing... apt-install wget

**Reply by Manu-65** [reply]
Thanks for your feedback :-)👍 
wget was installed.
yes, it is arm64.
i tried asylumexp script, it give the same output/result than tteck one
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/add-tailscale-lxc.sh)"
```
```
bash -c "$(wget -qLO - https://github.com/asylumexp/Proxmox/raw/main/misc/add-tailscale-lxc.sh)"
```

I think the problem is the linux version, i have installed linux debian 12 Bookworm.

https://tailscale.com/kb/1031/install-linux?q=linux

![imagen](https://github.com/user-attachments/assets/4a33e587-7267-4d7a-980d-ec8267b38ccc)

Shall Bullseye way forward?

**Reply by MickLesk** [reply]
Same issue with wget installed? 🤔

Install mc / sudo and curl and reinstall wget and Check again

**Reply by asylumexp** [reply]
The Tailscale upstream script is same as one from arm64 repo, as no changes were needed.

Try instead using Debian LXC created from [arm64 repo](https://pimox-scripts.com)

**Reply by Manu-65** [reply]
thanks, i picked up the next Debian LXC script:
```
bash -c "$(wget -qLO - https://github.com/asylumexp/Proxmox/raw/main/ct/debian.sh)"
````
i executed it in proxmox shell, but...

![imagen](https://github.com/user-attachments/assets/e35b48a5-700f-4052-8168-7cb61a065daa)


i found out the template in local storage, the scripts didn`t take in account "datastore" as destination

![imagen](https://github.com/user-attachments/assets/749aab76-2267-48ab-ac83-60c70c357ab4)

![imagen](https://github.com/user-attachments/assets/0f6eb8cb-4d2f-419e-ab40-f394951aaa3e)

i will try with "local" and not with "datastore"

Working!! Creating container:

![imagen](https://github.com/user-attachments/assets/e5c9563e-55c6-4fc8-bb4b-cb08afb6fe92)

Then
```
bash -c "$(wget -qLO - https://github.com/asylumexp/Proxmox/raw/main/misc/add-tailscale-lxc.sh)"
```
And finally... Ta-Chan!!

![imagen](https://github.com/user-attachments/assets/e19fff33-e601-4811-9888-10fa6a09979c)

Thanks very much to both of you!!

**Reply by asylumexp** [reply]
Oops, my bad wifi must have made that second message post late, since I originally made it right after I did the first one.

> i found out the template in local storage, the scripts didn`t take in account "datastore" as destination

When I’m able to do work again I’ll work on fixing this so template location does work.

@MickLesk / other maintainers, can you just @ me or tell people to leave their issues in [my repo](https://github.com/asylumexp/Proxmox), cause otherwise I have to keep checking to see if anyone makes an issue related to arm64 here.
